R/bfast.R

```diff
@@ -120,18 +123,18 @@ bfast <- function (Yt, h = 0.15, season = c("dummy", "harmonic", "none"),
     si3 <- sin(2 * pi * tl * w * 3)
     smod <- Wt ~ co + si + co2 + si2 + co3 + si3
     # Start the iterative procedure and for first iteration St=decompose result
-    if (decomp == "stlplus") {
-        St <- stlplus(Yt, t=ti, n.p = f, s.window = "periodic", ...)$data[, "seasonal"]
+    St <- if (decomp == "stlplus") {
+        stlplus::stlplus(Yt, t=ti, n.p = f, s.window = "periodic",   ...)$data[, "seasonal"]
```

Comment:
```suggestion
        stlplus::stlplus(Yt, t=ti, n.p = f, s.window = "periodic", ...)$data[, "seasonal"]
```
I am not sure if this was intentional, but I suppose one space is enough.